### ISSUE TITLE
autoHeight issue

### ISSUE THREAD

**Issue by jacklinwood** [webbiesdk/SudoSlider#65]
Hi mate,

I have a slider with autoHeight left as true. I then have a slider inside one of those slides. This seems to work okay apart from the autoHeight not working on the one inside. I have longer content in the second slide but that is cut off to the same height as the first slide until I change the browser viewport, then the rest of the content is shown.

Any ideas?

Thanks,

Jack

**Comment by webbiesdk** [comment]
I got an idea how to do it. 

Setting the option ````performanceMode: 1```` on the outer slider makes sure it such that the only the visible slide is actually shown in the DOM, meaning that autoheight doesn't have to be on while the outer slider is not animating. 
If you then add some callbacks to the inner slider, that makes sure to disable autoheight when the inner slider is animating, then everything works. 

I updated the previous example to show it. Just look at the source code: http://webbies.dk/tmp/nested/demos/nested.html 

The animations will behave weirdly if you try and animate the outer slider while the inner slide is animating. But it will just snap in place, the page won't break.
